app-frontend/src/app/components/toolItem/toolItem.html

```diff
@@ -7,13 +7,14 @@
        class="rounded-img item-img">
   <div>
     <div class="list-item-header">
-      {{$ctrl.toolData.name}}
+      {{$ctrl.toolData.title}}
     </div>
     <div class="list-item-description">
       {{$ctrl.toolData.description}}
     </div>
+    <!-- @TODO: This username will need to be dynamic once other users can create tools -->
     <div class="list-item-attribution">
-      Uploaded by: {{$ctrl.toolData.uploader}}
+      Uploaded by: Raster Foundry
```

Comment:
Hard-coding Raster Foundry here intentionally.

We're only provided with a user-id and don't have a convenient method to retrieve user details.